### PR TITLE
[cdap-686] ${cdap.site} variable added to cdap-site.xml

### PR DESCRIPTION
**Pull request by kpovetkin**
According to https://issues.cask.co/browse/CDAP-686

### CODE REVIEW

File: cdap-standalone/src/main/resources/cdap-site.xml [hunk 1]
```diff
@@ -14,6 +14,11 @@
   the License.
   -->
 <configuration>
+    <property>
```
Comment (kpovetkin) [inline cdap-standalone/src/main/resources/cdap-site.xml]: It seems to be standalone specific only. This property is required to simplify fully-qualified paths usage in config files.

File: cdap-standalone/src/main/resources/cdap-site.xml [hunk 2]
```diff
@@ -14,6 +14,11 @@
...
+        <name>cdap.site</name>
+        <value></value>
```
Comment (kpovetkin) [inline cdap-standalone/src/main/resources/cdap-site.xml]: @awholegunch Should it be changed?

**Comment by kpovetkin** [comment]
Updated according @chtyim comments.

File: cdap-common/src/main/resources/cdap-default.xml [hunk 3]
```diff
@@ -92,8 +92,14 @@
     </property>
 
     <property>
+        <name>user.dir</name>
```
Comment (chtyim) [inline cdap-common/src/main/resources/cdap-default.xml]: Better not to call it `user.dir` as it clash with the standard java system property. Call it something like `local.root.dir` and have the value as `${user.dir}`.

File: cdap-docs/admin-manual/source/installation/cdap-site.rst [hunk 4]
```diff
@@ -41,6 +41,9 @@ see the :ref:`configuration-security` section.
    * - ``app.temp.dir``
      - ``/tmp``
      - Temp directory
+   * - ``cdap.site``
```
Comment (chtyim) [inline cdap-docs/admin-manual/source/installation/cdap-site.rst]: This is not consistent with the property name.

**Comment by kpovetkin** [comment]
@chtyim Updated.

**Comment by awholegunch** [comment]
Closed this PR as it is no longer required.

**Pull request closed without merging**